angular_gallery_section/pubspec.yaml

```diff
@@ -7,10 +7,10 @@ environment:
   sdk: '>=2.0.0-dev.46.0 <2.0.0'
 dependencies:
   analyzer: ^0.31.1
-  angular: 5.0.0-alpha+11
-  angular_components: 0.9.0-alpha+11
+  angular: 5.0.0-alpha+12
+  angular_components: 0.9.0-alpha+12
   build: '>=0.11.1 <0.13.0'
-  build_config: ^0.2.5
+  build_config: ^0.2.6
   glob: ^1.1.5
   markdown: ^1.1.1
```

Comment:
bump this!

Reply:
This one is blocked on some internal changes. Will update with those are done.